iidm/iidm-extensions/src/test/java/com/powsybl/iidm/network/extensions/TwoWindingsTransformerPhaseAngleClockTest.java

```diff
@@ -35,10 +35,12 @@ public void setUp() {
 
     @Test
     public void testEnd() {
-        TwoWindingsTransformerPhaseAngleClock pac = new TwoWindingsTransformerPhaseAngleClock(transformer, 11);
+        TwoWindingsTransformerPhaseAngleClock pac = new TwoWindingsTransformerPhaseAngleClockImpl(transformer, 11);
         transformer.addExtension(TwoWindingsTransformerPhaseAngleClock.class, pac);
```

Comment:
Don't use `addExtension(TwoWindingsTransformerPhaseAngleClock.class, pac)` anymore, use the new method